sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Program.cs

```diff
@@ -80,25 +82,18 @@ public static async Task Main(string[] args)
         }
 
         /// <summary>
-        /// Illustrates how to construct a <see cref="DigitalTwinsClient"/>, using the <see cref="ClientSecretCredential"/>
+        /// Illustrates how to construct a <see cref="DigitalTwinsClient"/>, using the <see cref="DefaultAzureCredential"/>
         /// implementation of <see cref="Azure.Core.TokenCredential"/>.
-        /// </summary>
-        /// <param name="tenantId">The Id of the tenant of the application Id.</param>
-        /// <param name="clientId">The application Id.</param>
-        /// <param name="clientSecret">A client secret for the application Id.</param>
+        /// </summary>        
         /// <param name="adtEndpoint">The endpoint of the digital twins instance.</param>
-        private static DigitalTwinsClient GetDigitalTwinsClient(string tenantId, string clientId, string clientSecret, string adtEndpoint)
-        {
+        private static DigitalTwinsClient GetDigitalTwinsClient(string adtEndpoint)
+        {   
             #region Snippet:DigitalTwinsSampleCreateServiceClientWithClientSecret
 
-            // By using the ClientSecretCredential, a specified application Id can login using a
+            // By using the DefaultAzureCredential, a specified application Id can login using a
```

Comment:
You'd need to run snippets script for this change :)

Reply:
Good call :)